### PR TITLE
docs: enhance story of Bar, FlexBox, Grid and FlexibleColumnLayout

### PR DESCRIPTION
**Pull request by Lukas742**
**Thank you for your contribution!** 👏

To get it merged faster, kindly review the checklist below:

## Pull Request Checklist
- [x] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-webcomponents-react/blob/master/CONTRIBUTING.md)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-webcomponents-react/blob/master/CONTRIBUTING.md#contribute-code) section 
- [x] [Correct commit message style](https://github.com/SAP/ui5-webcomponents-react/blob/master/docs/Guidelines.md#commit-message-style)

### CODE REVIEW

File: packages/main/src/components/FlexBox/FlexBox.jss.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ export const styles = {
   flexBox: {
     display: 'flex'
   },
+  fitContainer: { width: '100%', height: '100%', backgroundColor: 'red' },
```
Comment (MarcusNotheis) [inline packages/main/src/components/FlexBox/FlexBox.jss.ts:5]: I guess the background color is a leftover?

File: packages/main/src/components/AnalyticalTable/__snapshots__/AnalyticalTable.test.tsx.snap [hunk 2]
```diff
@@ -32,7 +32,7 @@ exports[`AnalyticalTable Alternate Row Color 1`] = `
         style="width: 600px;"
       >
         <div
-          class="resizer-0-2-31"
+          class="resizer-0-2-32"
```
Comment (MarcusNotheis) [inline packages/main/src/components/AnalyticalTable/__snapshots__/AnalyticalTable.test.tsx.snap:35]: why do we even have rule counters here?

**Review by MarcusNotheis — CHANGES_REQUESTED** [review]

File: packages/main/src/components/FlexBox/FlexBox.jss.ts (hunk 1 above)
Comment (Lukas742) [reply packages/main/src/components/FlexBox/FlexBox.jss.ts]: Oops :sweat_smile:
It's removed now.

File: packages/main/src/components/AnalyticalTable/__snapshots__/AnalyticalTable.test.tsx.snap (hunk 2 above)
Comment (Lukas742) [reply packages/main/src/components/AnalyticalTable/__snapshots__/AnalyticalTable.test.tsx.snap]: There was no name specified in the `createUseStyles` options which was the reason for this behavior

**Review by MarcusNotheis — APPROVED** [review]

**Pull request merged**